docs/iviewwindowsprops-api-windows.md

```diff
@@ -9,6 +9,14 @@ This extends the [View Props](https://reactnative.dev/docs/view#props) and [`IKe
 
 ## Methods
 
+### `accessibilityLandmarkType`
+
+Specifies the UIA Landmark Type for the View. 
+
+| type | required |
+|:--|:--|
+| string | No |
+
 ### `accessibilityPosInSet`
```

Comment:
That's a core prop. Do we have a way of only documenting the added "windows-only" values we add to the enum?

Reply:
If this is in reference to accessibilityRole="header" that's an RN-core enum value, we just needed additional logic to get the same behavior.

Reply:
that's fair. It would be good to know what APIs are supported on Windows or not. I see we have an API completion task for that property but it's still marked active (probably because the issue also mentions accessibilityState?)
In other words, should this task be active still, or split into two and one of them closed?
https://github.com/microsoft/react-native-windows/issues/8191

Reply:
@asklar I think that task should still be active, as we don't actually implement setting the AutomationControlType which I think was the goal for implementing accessibilityRole for Text. Currently, we're just setting the localizedControlType since that can be done without subclassing, so we might want to keep that work item alive unless we decide that setting LocalizedControlType is enough. Although splitting the two might still make sense... they're only related in that both would need the same workaround to get fully working for Text/TextBlock.